apps/web/src/app/(private)/evaluations/(evaluation)/[evaluationUuid]/editor/_components/EvaluationEditor/Editor/Playground/Variables/components/CopyButton.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button, Icon, toast } from '@latitude-data/web-ui'
+
+export const CopyButton = ({ text }: { text: string }) => (
```

Comment:
This is a molecule. Is fully generic

Reply:
There's already a `ClickToCopy` molecule that is just that

Reply:
we will consolidate in a future PR